src/app/core/basic-datatypes/entity/display-entity/display-entity.component.scss

```
@@ -1,13 +1,7 @@
-@use "variables/sizes";
+@use "../../../../../styles/variables/sizes";
```

Is this intended?

---

```suggestion
@use "variables/sizes";
```
no, I don't think so ...